javascript/ql/test/query-tests/Statements/UseOfReturnlessFunction/tst.js

```diff
@@ -79,4 +79,7 @@
 	import { filter } from 'lodash'
 	var bar = filter([1,2,4], x => { equals(x, 3) } ) // NOT OK!
 	console.log(bar);
+	
+	var baz = [1,2,3].filter(n => {n === 3}) // OK
```

Comment:
Why is this OK?

Reply:
Because the `js/useless-expression` flags the `n === 3` expression, and by not marking this line we avoid getting two reports for the same error.